packages/web-frontend/src/containers/Map/AreaTooltip.js

```diff
@@ -34,8 +41,76 @@ export class AreaTooltip extends Component {
     }
   }
 
+  getTextList() {
+    const { orgUnitName, hasMeasureValue, measureOptions, orgUnitMeasureData } = this.props;
+
+    const defaultTextList = [
+      <Heading key={0} hasMeasureValue={hasMeasureValue}>
+        {orgUnitName}
+      </Heading>,
+    ];
+
+    const renderTextList = (data, textList = []) => {
+      Object.keys(data).forEach(key => {
+        textList.push(<span key={key}>{`${key}: ${data[key]}`}</span>);
+      });
+      return textList;
+    };
+
+    if (hasMeasureValue) {
+      const legacy = measureOptions.length === 1;
+      const { formattedMeasureData, onlyOneValue } = legacy
+        ? this.buildLegacyFormattedMeasureData(orgUnitMeasureData, measureOptions)
+        : this.buildFormattedMeasureData(orgUnitMeasureData, measureOptions);
+
+      if (onlyOneValue) {
+        return renderTextList({ [orgUnitName]: formattedMeasureData });
+      }
+
+      renderTextList(formattedMeasureData, defaultTextList);
+    }
+
+    return defaultTextList;
+  }
+
+  buildLegacyFormattedMeasureData(orgUnitMeasureData, measureOptions) {
+    const formattedValue = getSingleFormattedValue(orgUnitMeasureData, measureOptions);
+
+    return {
+      formattedMeasureData: formattedValue,
+      onlyOneValue: true,
+    };
+  }
+
+  buildFormattedMeasureData(orgUnitMeasureData, measureOptions) {
+    const formattedMeasureData = {};
+
+    if (orgUnitMeasureData) {
+      formattedMeasureData[measureOptions[0].name] = getSingleFormattedValue(
+        orgUnitMeasureData,
+        measureOptions,
+      );
+
+      measureOptions.forEach(({ key }) => {
+        if (key !== measureOptions[0].key) {
+          formattedMeasureData[key] = orgUnitMeasureData[key];
+        }
+      });
+    }
+
+    if (Object.values(formattedMeasureData).length === 1) {
+      return {
+        formattedMeasureData: Object.values(formattedMeasureData)[0],
+        onlyOneValue: true,
+      };
```

Comment:
I'm actually kinda on the fence about this behaviour, leaning towards us always showing it as `<measureName>: <value>` even if there's only a single measure value for this particular org unit.

Assuming that there's more than 1 measureOption (which is the case by the time we call this method) then this is a multi-value map overlay. In which I think the user needs to know which value is which.

eg. For Vic there is just a value for `Respondents`, whereas For NSW there's just a value for `% with flu`
I'd lean toward showing:
```
Vic
Respondents: 7
```
and
```
NSW
% with flu: 6%
```

instead of:
```
Vic: 7
```
and
```
NSW: 6%
```

What do ya think?

Reply:
Current system use either only `orgUnit` or `orgUnit:value`. I think yeah there are too much conditions running here (even though just 3 including the list we are adding). We can reduce to two which are:

```
Vic (only orgUnit)
```

```
Vic (orgUnit with list)
Respondents: 7
...
```

Reply:
I think I agree with you 👍 but lets have a quick chat to make sure we're on the same page?